scripts/qtb

```diff
@@ -33,8 +33,8 @@ def main():
             parser.error("Please specify widget name and argument.")
         try:
             client.widget.__getitem__(args[0]).update(" ".join(args[1:]))
-        except command.CommandError, v:
-            print >> sys.stderr, v
+        except command.CommandError as v:
+            print >>sys.stderr, v
```

Comment:
Oh, interesting, something that got missed in the Python 3 move, I can take care of that seperately